test/src/k-provider/ovp/media-config-data.js

```diff
@@ -35,7 +35,7 @@ const NoPluginsNoDrm = {
       HLSOnly: 'android',
       ChannelName: 'Disney Channel SE',
       tags: '',
-      heightRatio: 0.56
+      aspectRatio: 1.78
```

Comment:
might be we can use const here instead of hardcoded value, since it repeats couple of times in same file